src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's street in the address book.
+ */
+
+public class PostalCode {
+
+    private String postal;
+
+    public PostalCode(String block) {
```

Comment:
Why does the constructor for `PostalCode` accept a `block` as an argument :P Same for `Street` and `Unit`...

Remember to change code properly when copying code snippets!